src/blocks/content-helper/current-post-details/post-performance-data.ts

```diff
@@ -0,0 +1,29 @@
+export interface PostPerformanceData {
+	author: string;
+	avgEngaged: string;
+	date: string;
+	id: number;
+	period: {
+		start: string,
+		end: string,
+		days: number,
```

Comment:
Maybe a lint rule to enforce consistency
```suggestion
		start: string;
		end: string;
		days: number;
```